packages/agent/src/agent/http/errors.ts

```diff
@@ -2,9 +2,27 @@ import { AgentError } from '../../errors';
 import { HttpDetailsResponse } from '../api';
 
 export class AgentHTTPResponseError extends AgentError {
-  constructor(message: string, public readonly response: HttpDetailsResponse) {
+  constructor(
+    message: string,
+    public readonly response: HttpDetailsResponse,
+  ) {
     super(message);
     this.name = this.constructor.name;
     Object.setPrototypeOf(this, new.target.prototype);
   }
 }
+
+export class AgentCallError extends AgentError {
+  constructor(
+    message: string,
+    public readonly response: HttpDetailsResponse,
+    public readonly requestId: string,
```

Comment:
Is it intentional that this type has some fields snake_case and one field camelCase?

Reply:
fair point